service/s3/transport/http/response_error.go

```diff
@@ -0,0 +1,33 @@
+package http
```

Comment:
Same comment as host_id.go file, not really http specific, but higher level concept. maybe better under `s3metadata`, or just `metadata` too.